nouvelles-perspectives-en-sciences-sociales.csl

```diff
@@ -2,17 +2,19 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" initialize="false" demote-non-dropping-particle="never" default-locale="fr-CA">
   <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Nouvelles Perspectives des Sciences Sociales (French - Canada)</title>
+    <title>Nouvelles perspectives en sciences sociales (French - Canada)</title>
```

Comment:
approved

Reply:
Approved

Reply:
We approve this change. The correct title is : Nouvelles perspectives en sciences sociales